lib/client-assets.php

```diff
@@ -190,7 +190,7 @@ function gutenberg_register_packages_scripts() {
 		// Replace `.js` extension with `.asset.php` to find the generated dependencies file.
 		$asset_file   = substr( $path, 0, -3 ) . '.asset.php';
 		$asset        = file_exists( $asset_file )
-			? include_once( $asset_file )
+			? require_once( $asset_file )
```

Comment:
require once does the same thing as include once here right? we're still only including it once

Reply:
yes, it has a stronger contract but works the same otherwise